package.json

```diff
@@ -2,27 +2,28 @@
   "name": "solid-client",
   "version": "0.22.4",
   "description": "Common library for writing Solid read-write-web applications",
-  "main": "./index.js",
+  "main": "./lib/index.js",
   "files": [
-    "index.js",
     "config.js",
     "lib",
-    "dist"
+    "dist",
+    "src"
```

Comment:
No prob with removing it. I saw that we included the src dir in other libs, but maybe the thing to do is to not include it anywhere.